src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json

```diff
@@ -20,7 +20,8 @@
     "Microsoft.DotNet.xunit.performance.run.core": "99.99.99-dev",
     "Microsoft.NETCore.DotNetHostPolicy":"1.0.1-rc2-002357-00",
     "NETStandard.Library": "1.5.0-rc2-23931",
-    "xunit.extensibility.execution": "2.1.0"
+    "Microsoft.xunit.extensibility.execution.netcore": "1.0.0-prerelease",
```

Comment:
Why is it that we need to replace xunit.extensibility.execution?  I get worried about replacing external dependencies with ones that we create.